README.md

```diff
@@ -127,10 +131,31 @@ Please install the ```tree``` command before moving forward:
         ```
 
    - MacOSX
-        ```
-        install tree - http://apple.stackexchange.com/questions/128876/whats-the-equivalent-of-the-unix-tree-command-on-osx
-        install wget - http://stackoverflow.com/questions/4572153/os-x-equivalent-of-linuxs-wget
-        ```
+       
+        - Option 1:
+        
+          - install tree
+          
+          http://apple.stackexchange.com/questions/128876/whats-the-equivalent-of-the-unix-tree-command-on-osx
+        
+          - install wget 
+          
+          http://stackoverflow.com/questions/4572153/os-x-equivalent-of-linuxs-wget
+       
+        - Option 2 (Via brew ):
```

Comment:
That's great, if you can put this as a side note around this para, then the next person knows why we are doing this - atm lots of new suggestions are coming but its not getting communicated to the new or existing users.

This is for everyone's benefit, leave as much info as a side note as possible.